### ISSUE TITLE
Second (non first) menu-item not selectable (Quark 1.0.2)

### ISSUE THREAD

**Issue by awrog** [getgrav/grav-theme-quark#9]
Second (non first) menu-item not selectable (Quark 1.0.2)
Reproducible [here](http://cantorij.net): menu Seizoen, second item is not selectable (this is also the case for the sub-items). See also below
![quark-menu](https://user-images.githubusercontent.com/12153667/36531376-e80a7a7e-17bd-11e8-89f5-6aa6f7851801.gif)

**Comment by tkduman** [comment]
It looks like not the 2nd but below the separator line. As your cursor goes below the line menu disappear.

**Comment by awrog** [comment]
Can it be fixed?

**Comment by rhukster** [comment]
I'm not really able to replicate this, I can navigate just fine:

![screen capture on 2018-03-09 at 16-35-46](https://user-images.githubusercontent.com/1084697/37234928-f9bb22e0-23b7-11e8-8037-419e952c8fad.gif)

**Comment by kiranetph** [comment]
It seems it is related to Grav v1.3.10
See : https://discourse.getgrav.org/t/quark-drop-down-menus-issue/5829

**Comment by djdakta** [comment]
Hi,
It's indeed related to Grav v1.3.10.

**Comment by doansson** [comment]
Still happening with Grav v1.4

**Comment by awrog** [comment]
Updated GRAV (1.4.1) and Quark (1.0.2) to latest versions.
The problem still exists.
Tried Win 10 Pro (1920 x 1080) and Win 7 enterprise (1680 x 1050)
Tried Chrome (65 64-bit), FF (59 64-bit) and Edge (41)
You can try it [here](http://www.cantorij.net)
Screencast: Chrome, FF and Edge
![quark-menu](https://user-images.githubusercontent.com/12153667/37449689-833ba5d6-282b-11e8-88d3-49b468efd9b6.gif)

**Comment by rpowers** [comment]
I am having this issue as well with 1.4.2 and Safari, Chrome.

**Comment by awrog** [comment]
@rpowers This does the trick!
Thnx!

**Comment by tkduman** [comment]
What you do is when you're hovering navigation, setting it's Z index to 100. (Possibly above everything.)

Due to this while hovering the navigation, it'll be always on top. I see nothing wrong with it.

**Comment by tremel** [comment]
When using maps generated by [leaflet](https://leafletjs.com) by a script the menus are behind the maps. A z-index set in the theme would be generally useful.

**Comment by pamtbaau** [comment]
Hovering the submenu works fine to me in Quark, however, I do have the same problem when using an inherited theme.

**Comment by Goutte** [comment]
To reproduce:

1. inherit the quark theme
2. create a template extending the base, say `mypage.html.twig`:
```twig
{% extends 'partials/base.html.twig' %}

{% block content %}
    {{ page.content|raw }}
{% endblock %}
```

On this page the menu will bug. If you extend modular instead, it works, that's perhaps why you don't see it?

Not 100% sure about these instructions, as I have mathjax, markdown-notices and diagrams plugins enabled, and they may be culprits as well.

---

Thanks @rpowers for the charming fix!

**Comment by pamtbaau** [comment]
@rhukster On the forum someone experienced this issue again. Did some research and can reproduce the issue.

Steps to reproduce:
- Fresh Grav installation v1.6.26
- Add the following folder:
  ```
  pages
  ├── 01.home
  │   └── default.md
  ├── 02.typography
  │   └── default.md
  └── 03.topmenu
      ├── 01.sub1
      │   └── default.md
      ├── 02.sub2
      │   ├── 01.subsub1
      │   │   └── default.md
      │   ├── 02.subsub2
      │   │   └── default.md
      │   └── default.md
      └── default.md
  ```
- In quark.yaml set `header-fixed: false`
- Result in browser
![disappear](https://user-images.githubusercontent.com/5120011/88718406-36ff0f80-d122-11ea-89d2-95dafdb063d3.gif)
- In quark.yaml set `header-fixed: true`
- Result in browser
![remain](https://user-images.githubusercontent.com/5120011/88718835-a543d200-d122-11ea-9465-316d4f7025a3.gif)

**Comment by StevenBrons** [comment]
I have the same issue here; does not seem browser dependent.
Setting the "z-index" of #header is a possible fix.

**Comment by pamtbaau** [comment]
@StevenBrons, Yes, I know. It has been  mentioned above. But I don't think the `z-index` is a possible fix, but rather a possible workaround... ;-)

**Comment by awrog** [comment]
The problem is still there (GRAV grav-admin-v1.7.48 / Quark theme grav-theme-quark-2.1.0)
First you have to set 
```
dropdown:
  enabled: false
```
in quark.yaml
Would be nice to have something like

![Image](https://github.com/user-attachments/assets/6ee0db38-3c4f-45a3-8763-7c2de62d65cb)

In quark.yaml set `header-fixed: false`
Result in browser

![Image](https://github.com/user-attachments/assets/99650d2d-f7dd-4cbb-b0cc-10f13c1937ab)

In quark.yaml set `header-fixed: true`
Result in browser

![Image](https://github.com/user-attachments/assets/e0f3f386-1de7-4d01-a9fa-d07dd562222b)

**Comment by pmoreno-rodriguez** [comment]
Hello. I'm sorry I didn't address you. Many Grav users who opened issues several years ago are no longer here.

I'll take a look at your proposal as soon as possible
